src/test/regress/output/multi_alter_table_statements.source

```diff
@@ -552,6 +552,95 @@ SELECT master_create_worker_shards('single_shard_items', 1, 2);
  
 (1 row)
 
+-- Verify that ALTER TABLE .. REPLICATION IDENTITY [USING INDEX]* .. works
```

Comment:
One more test could be creating a Postgres table, creating unique index, setting replica identity and then creating the distributed table to see that Citus honors the replica identity on distributed table creation.

Reply:
Let's leave this for #1733